dbt_project.yml

```diff
@@ -66,6 +66,11 @@ vars:
     # Variables - Optional Consent model only
     snowplow__consent_cmp_visible: "{{ source('atomic', 'com_snowplowanalytics_snowplow_cmp_visible_1') }}"
     snowplow__consent_preferences: "{{ source('atomic', 'com_snowplowanalytics_snowplow_consent_preferences_1') }}"
+    # Variables - Optional Conversions only
+    # snowplow__conversion_events: [{'name': 'view_page', 'condition': "event_name = 'page_view'", "list_events": true},
```

Comment:
I think we may need to consider a property that takes into account how the conversion should be counted within a given context (once per event or once per session - I don't think we need once per user).

The most important context is probably within a session i.e., should we count a type of conversion more than once in a session, or just a single time? GA has just introduced the 'legacy' counting method of once per session: https://support.google.com/analytics/answer/13366706

Reply:
Would we want this to overwrite the `_volume` and `_total` values? Currently they could get it from just a case when >0 and accessing the first value array element, but can do that to overwrite those fields if it makes more sense